api/applicant/controllers/applicant.js

```diff
@@ -41,12 +48,32 @@ module.exports = {
       commitment,
       accepted,
       reason,
-      project: applicantProject.id,
+      project: applicantProject ? applicantProject : null,
       level
     };
 
     const entity = await strapi.services.applicant.create(application);
 
+    if (applicantProject){
+      const team = applicantProject.team;
+      console.log(team);
+      if(isNotEmpty(team.leaders)){
+        //lets leaders join google group
+        team.leaders.forEach(async (leader) => {
+          try {
+            const id = leader.leader.id ? leader.leader.id : leader.leader;
+            const email = leader.leader.email ? leader.leader.email: null;
+            console.log(email)
+            // Send an email to project lead.
+            await strapi.services.sendmail.send('kuikuigaocn@gmail.com', email, 'New applicant notificaton', `Hi Leaders, ${application.name} joined the project of "${applicantProject.title}".`);
```

Comment:
The first email needs to be the same as the email of the `transporter` right? Can we get it from the `transporter`?

Reply:
Yes, it's the same as the email of the transporter. I don't know how to get it from transporter, but we can get it from env variables after the email setting is done.

Reply:
Sounds good to me.